src/components/Venda.vue

```diff
@@ -271,7 +260,7 @@ export default {
       if (this.toasts) {
         this.toasts.close();
       }
-      if (defined(this.venda.sacolao) || defined(this.venda.pagamento)) {
+      if (this.venda.sacolao || defined(this.venda.pagamento)) {
```

Comment:
Na função de submit, para criar uma venda, atualmente é necessário selecionar o sacolão **OU** o método de pagamento.
Não deveria ser necessário selecionar o sacolão **E** o método de pagamento?

Reply:
oi @lgomesp é que inicialmente era apenas para gerir entregas de sacolão, porem agora ele gerencia entregas no geral, entao quando for somente entrega de compra ele nao obriga a selecionar o sacolão :P

Reply:
Entendi. Beleza 😄